{"name": "输入错误username、password，验证用户登录失败", "status": "passed", "steps": [{"name": "登陆case", "status": "passed", "parameters": [{"name": "host", "value": "'http://47.96.25.229:7300/mock/60e29c972b9d72001731a419/test'"}, {"name": "username", "value": "'admin1'"}, {"name": "password", "value": "'111111'"}], "start": 1626082567587, "stop": 1626082567624}], "attachments": [{"name": "log", "source": "e9416079-04c0-49b3-88fd-8c5fc9218e47-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_input", "value": "{'username': 'admin1', 'password': '111111'}"}, {"name": "data_expect", "value": "{'code': 1, 'data': 'login failed'}"}, {"name": "title", "value": "'输入错误username、password，验证用户登录失败'"}], "start": 1626082567587, "stop": 1626082567624, "uuid": "65fb2638-a3ff-4b92-a053-24073accda1e", "historyId": "01c6d545764316f586a22fa694ce7557", "testCaseId": "4cb0134c279d13d487923af9db52d091", "fullName": "case.test_login.TestLogin#test_login", "labels": [{"name": "feature", "value": "TestLogin"}, {"name": "story", "value": "登录接口测试用例"}, {"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "asce"}, {"name": "thread", "value": "28820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login"}]}